### PR TITLE
console lof removed

### PR DESCRIPTION
**Pull request by abdul5890575**

### CODE REVIEW

File: examples/index.html [hunk 1]
```diff
@@ -71,6 +72,57 @@
...
+        name: "Analysis",
+        data: [{x: 0, y: 2.0, color:'blue'}, {x: 1, y: 5.0, color:'blue'}, {x: 2, y: 6.0, color:'blue'}, {x: 3, y: 3.0, color:'orange'}, {x: 4, y: 4.0, color:'blue'},{x: 5, y: .70, color:'blue'}],
+        type: 'line'
+      },
+      {
+        name: "Additional",
+        data: [{x: 0, y: 5.0, color:'orange'}, {x: 1, y: 5.0, color:'brown'}, {x: 2, y: 4.0, color:'green'}, {x: 3, y: .80, color:'purple'}, {x: 4, y: .70, color:'red'},{x: 5, y: .70, color:'green'}],
+        type: 'line'
+      }]
+      new Chartkick.ColumnChart("Multi", [["Data and Privacy",2.5],["Internal Security",2.385],["Organizational security",1.667],["Product Security",2.7],["Infastructure security",1.379],["Physical Security",3]], {Datas: lineDataType4, combineCharts : true, format: false});
```
Comment (babarajay) [inline examples/index.html:110]: remove 'Datas' and add something meaningful for example: line_data

File: src/adapters/chartjs.js [hunk 2]
```diff
@@ -543,7 +543,7 @@ export default class {
     this.library = library;
   }
 
-  renderLineChart(chart, chartType) {
+  renderLineChartFromChartjs(chart, chartType) {
```
Comment (babarajay) [inline src/adapters/chartjs.js:546]: are we still using charts from ChartJs? I can see function name render LineChart from ChartJS? can you explain?

**Review by babarajay — CHANGES_REQUESTED** [review]

File: src/adapters/chartjs.js (hunk 2 above)
Comment (abdul5890575) [reply src/adapters/chartjs.js:546]: no we are not, there was already a linechart in there created by you, i just changed the name cuz chart js adapter is higher up and it starts running stuff from your chartjs first

File: examples/index.html (hunk 1 above)
Comment (abdul5890575) [reply examples/index.html:110]: ok

**Pull request closed without merging**